examples/vector_fe/vector_fe_ex8/vector_fe_ex8.C

```diff
@@ -327,7 +327,7 @@ compute_enriched_soln(const MeshBase & mesh,
   const auto & scalar_phi_face = scalar_fe_face.get_phi();
   const auto & enriched_scalar_phi_face = enriched_scalar_fe_face->get_phi();
   const auto & lambda_phi_face = lambda_fe_face.get_phi();
-  const auto & JxW_face = vector_fe_face.get_JxW();
+  const auto & JxW_face = scalar_fe_face.get_JxW();
```

Comment:
Why are these changes required?

Reply:
I did this to make sure the JxW we're getting from the XYZ map is accurate